README.md

```diff
@@ -12,68 +12,129 @@ SPDX-License-Identifier: MIT
+   @Environment(SpeechSynthesizer.self) private var speechSynthesizer
+   // The transcribed message from the user's voice input.
+   @State private var message = ""
+
+
+   var body: some View {
+      VStack {
+         // Button used to start and stop recording by triggering the `microphoneButtonPressed()` function.
+         Button("Record") {
+            microphoneButtonPressed()
+         }
+            .padding(.bottom)
+
+
+         // Button used to start and stop playback of the transcribed message by triggering the `playbackButtonPressed()` function.
+         Button("Playback") {
+            playbackButtonPressed()
+         }
+      }
+   }
+
+
+   private func microphoneButtonPressed() {
+      if speechRecognizer.isRecording {
+         // If speech is currently recognized, stop the transcribing.
+         speechRecognizer.stop()
+      } else {
+         // If the recognizer is idle, start a new recording.
+         Task {
+            do {
+               // The `speechRecognizer.start()` function returns an `AsyncThrowingStream` that yields the transcribed text.
+               for try await result in speechRecognizer.start() {
+                  // Access the string-based result of the transcribed result.
+                  message = result.bestTranscription.formattedString
+               }
+            }
+         }
+      }
+   }
+    
+   private func playbackButtonPressed() {
+      if speechSynthesizer.isSpeaking {
+         // If speech is currently synthezized, pause the playback.
+         speechSynthesizer.pause()
+      } else {
+         // If synthesizer is idle, start with the text-to-speech functionality.
+         speechSynthesizer.speak(message)
+      }
+   }
+}
 ```
 
-For more information, please refer to the API documentation.
-
-
 ## Contributing
 
 Contributions to this project are welcome. Please make sure to read the [contribution guidelines](https://github.com/StanfordSpezi/.github/blob/main/CONTRIBUTING.md) and the [contributor covenant code of conduct](https://github.com/StanfordSpezi/.github/blob/main/CODE_OF_CONDUCT.md) first.
 
-
 ## License
+This project is licensed under the MIT License. See [Licenses](https://github.com/StanfordSpezi/SpeziSpeech/tree/main/LICENSES) for more information.
+
 
-This project is licensed under the MIT License. See [Licenses](https://github.com/StanfordSpezi/SpeziContact/tree/main/LICENSES) for more information.
+## Contributors
+This project is developed as part of the Stanford Byers Center for Biodesign at Stanford University.
+See [CONTRIBUTORS.md](https://github.com/StanfordSpezi/SpeziSpeech/tree/main/CONTRIBUTORS.md) for a full list of all SpeziSpeech contributors.
 
 ![Spezi Footer](https://raw.githubusercontent.com/StanfordSpezi/.github/main/assets/FooterLight.png#gh-light-mode-only)
 ![Spezi Footer](https://raw.githubusercontent.com/StanfordSpezi/.github/main/assets/FooterDark.png#gh-dark-mode-only)
```

Comment:
This is a uniform header across all Spezi Modules, it would be great if we can keep it the same across all repos: https://swiftpackageindex.com/stanfordspezi/spezi/documentation/spezi/documentation-guide#Footer.